File: developer/Samples/NET 4.6/WF_WCF_Samples/WCF/Basic/Binding/ByteStreamEncoder/VB/ByteStreamHttpBinding/ByteStreamHttpDefaults.vb
'----------------------------------------------------------------
' Copyright (c) Microsoft Corporation.  All rights reserved.
'----------------------------------------------------------------

Imports Microsoft.VisualBasic
Imports System
Imports System.ServiceModel
Imports System.Xml

Namespace Microsoft.Samples.ByteStreamEncoder

    Friend NotInheritable Class ByteStreamHttpDefaults

        Friend Const DefaultBypassProxyOnLocal As Boolean = True

        Friend Const DefaultHostNameComparisonMode As HostNameComparisonMode = HostNameComparisonMode.StrongWildcard

        Friend Const DefaultMaxBufferPoolSize As Int64 = 512 * 1024

        Friend Const DefaultMaxBufferSize As Int32 = 65536

        Friend Const DefaultMaxReceivedMessageSize As Int64 = 65536

        Friend Const DefaultProxyAddress As String = Nothing

        Friend Const DefaultReaderQuotas As String = Nothing

        Friend Const DefaultTransferMode As TransferMode = TransferMode.Streamed

        Friend Const DefaultUseDefaultWebProxy As Boolean = True

    End Class
End Namespace

File: developer/Samples/NET 4.6/WF_WCF_Samples/WCF/Basic/Binding/ByteStreamEncoder/CS/ByteStreamHttpBinding/ByteStreamHttpDefaults.cs
//----------------------------------------------------------------
// Copyright (c) Microsoft Corporation.  All rights reserved.
//----------------------------------------------------------------

using System;
using System.ServiceModel;
using System.Xml;

namespace Microsoft.Samples.ByteStreamEncoder
{

    static class ByteStreamHttpDefaults
    {
        internal const Boolean DefaultBypassProxyOnLocal = true;
        internal const HostNameComparisonMode DefaultHostNameComparisonMode = HostNameComparisonMode.StrongWildcard;
        internal const Int64 DefaultMaxBufferPoolSize = 512 * 1024;
        internal const Int32 DefaultMaxBufferSize = 65536;
        internal const Int64 DefaultMaxReceivedMessageSize = 65536;
        internal const Uri DefaultProxyAddress = null;
        internal const XmlDictionaryReaderQuotas DefaultReaderQuotas = null;
        internal const TransferMode DefaultTransferMode = TransferMode.Streamed;
        internal const Boolean DefaultUseDefaultWebProxy = true;
    }
}
